lib/gssapi/mech/gss_mech_switch.c

```diff
@@ -162,11 +162,13 @@ _gss_string_to_oid(const char* s, gss_OID *oidp)
 }
 #endif
 
+typedef void (*f_ptr)();
```

Comment:
I feel that the introduction and use of f_ptr should be a separate change with a commit message explaining what warning messages it is suppressing.

Reply:
Hmm, maybe I can rework this one and avoid the typedef.